code/infrastructure-api/src/kubeWatcher/kubeWatcher.js

```diff
@@ -45,16 +45,14 @@ export function podReadyWatcher(event) {
 
   let output = Promise.resolve();
 
-  if (event.status.phase === 'Running' && event.metadata.deletionTimestamp === undefined) {
-    if (find(event.status.conditions, { type: 'Ready', status: 'True' })) {
-      logger.debug(`Pod ready -- name: "${kubeName}", type: "${type}"`);
-
-      if (stackNames.includes(type)) {
-        // Minio containers, like Stacks, are tagged with 'user-pod' but are not recorded in stacks DB. Only Stacks should
-        // have their status updated.
-        output = stackRepository.updateStatus({ name, type, status: READY })
-          .then(() => logger.debug(`Updated status record for "${name}" to "${READY}"`));
-      }
+  if (isPodRunning(event)) {
+    logger.debug(`Pod ready -- name: "${kubeName}", type: "${type}"`);
+
+    if (stackNames.includes(type)) {
```

Comment:
You could even pull out this additional `if` into the method. Happy for it to stay like this though so will approve and let you decide.